### PR TITLE
BD-2229: Updates to Shopify multiple store support

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_hidden/private_betas/multiple_stores.md [hunk 1]
```diff
@@ -7,19 +7,19 @@ layout: dev_guide
 
 # Shopify multiple store support
 
-> Connect multiple Shopify stores to a single app group with our new beta for multiple store support to have a holistic view of your customers across all markets or brands. Build and launch automation programs and journeys in a single workspace without duplicating efforts across multiple instances. 
+> Connect multiple Shopify stores to a single app group with our new beta for Multiple Store Support to have a holistic view of your customers across all markets or brands. Build and launch automation programs and journeys in a single workspace without duplicating efforts across multiple instances.
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/multiple_stores.md:10]:
```suggestion
> Connect multiple Shopify stores to a single app group with our new beta for multiple store support to have a holistic view of your customers across all markets or brands. Build and launch automation programs and journeys in a single workspace without duplicating efforts across multiple instances. 
```

File: _docs/_hidden/private_betas/multiple_stores.md [hunk 2]
```diff
@@ -33,30 +33,39 @@ With Braze’s multiple-store support, you can:
...
 3. For the collect email or sms subscribers step:
-- **Customers with multiple regions and markets as stores**: Ensure you set the correct email subscription group to the right store for every store install. We recommend you enable “Override existing global subscription state for users” for every store. This will always override the Braze global email subscription state and the selected email subscription group with the most recent subscription from any Shopify store for the user. 
-- **Customers with multiple brands as stores**: Currently, we do not have support to override only the email subscription group (recommended). This will be supported ahead of general availability.<br><br> 
-4. Repeat this process for each store as many times as you want.<br><br>
+- **Customers with multiple regions and markets as stores**: Ensure you set the correct email subscription group to the right store for every store install. To track a user's most recent subscription state from any store, you must enable the “Override existing global state for users” option when installing each store. Doing so will override the Braze global email subscription state and the Email Subscription group.
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/multiple_stores.md:36]:
```suggestion
- **Customers with multiple regions and markets as stores**: Ensure you set the correct email subscription group to the right store for every store install. To track a user's most recent subscription state from any store, you must enable the “Override existing global state for users” option when installing each store. Doing so will override the Braze global email subscription state and the email subscription group.
```

File: _docs/_hidden/private_betas/multiple_stores.md [hunk 3]
```diff
@@ -33,30 +33,39 @@ With Braze’s multiple-store support, you can:
...
 
-### Shopify user merging
-If you have the same user across multiple Shopify stores, Braze syncs customer data to a single profile if their email address or phone number already exists using the alias, `shopify_email` or `shopify_phone`. Note that it is possible for user data (i.e., city) to not match across multiple Shopify stores for the same user, but Braze will always update the user profile with the most recent activity on any store. 
+### Shopify user merging and syncing
+
+If the user’s Shopify Customer ID, email address, or phone number exists already within Braze using the alias, {% raw %}`shopify_customer_id_{{storefront_domain}}` `shopify_email`, or `shopify_phone`, {% endraw %} then we’ll update the existing user profile. If those aliases do not exist within Braze, we will create a new user profile. Note that it is possible for a user’s data (e.g., city) to differ across multiple Shopify stores for the same user; in such cases, Braze will always update the user profile from the store with the most recent activity.
```
Comment (KellieHawks) [inline _docs/_hidden/private_betas/multiple_stores.md:60]:
```suggestion
If the user’s Shopify customer ID, email address, or phone number exists already within Braze using the alias, {% raw %}`shopify_customer_id_{{storefront_domain}}` `shopify_email`, or `shopify_phone`, {% endraw %} then we’ll update the existing user profile. If those aliases do not exist within Braze, we will create a new user profile. Note that it is possible for a user’s data (e.g., city) to differ across multiple Shopify stores for the same user; in such cases, Braze will always update the user profile from the store with the most recent activity. 
```

File: _docs/_hidden/private_betas/multiple_stores.md [hunk 4]
```diff
@@ -10,16 +10,16 @@ layout: dev_guide
...
-| Audit and update Shopify alias | The `shopify_customer_id` alias will be migrated to `shopify_customer_id_{{storename}}` after you connect more than one store. Ensure you are updating any internal systems to use the new alias. The legacy alias, `shopify_customer_id`, will be deprecated. If you are not using the alias today, you can ignore this. |
+| Audit and update segments, campaigns, and Canvas using Shopify attributes | The custom attributes collected from Multiple Stores will be in the format of a nested object, which differs from the current structure used in the overall Shopify integration, which is formatted as a string value. As a result, you'll need to update all segments, campaigns, or Canvas to the new format after connecting multiple stores to use the “Nested Custom Attribute” filter or update the “Change Custom Attribute” trigger event.<br><br>If you are not using any of the attributes today, you can ignore this. |
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/multiple_stores.md:21]:
```suggestion
| Audit and update segments, campaigns, and Canvas using Shopify attributes | The custom attributes collected from multiple stores will be in the format of a nested object, which differs from the current structure used in the overall Shopify integration, which is formatted as a string value. As a result, you'll need to update all segments, campaigns, or Canvas to the new format after connecting multiple stores to use the “Nested Custom Attribute” filter or update the “Change Custom Attribute” trigger event.<br><br>If you are not using any of the attributes today, you can ignore this. |
```
I think this should be lowercased? 🤔

File: _docs/_hidden/private_betas/multiple_stores.md [hunk 5]
```diff
@@ -33,30 +33,39 @@ With Braze’s multiple-store support, you can:
...
 
-### Shopify user merging
-If you have the same user across multiple Shopify stores, Braze syncs customer data to a single profile if their email address or phone number already exists using the alias, `shopify_email` or `shopify_phone`. Note that it is possible for user data (i.e., city) to not match across multiple Shopify stores for the same user, but Braze will always update the user profile with the most recent activity on any store. 
+### Shopify user merging and syncing
+
+If the user’s Shopify customer ID, email address, or phone number exists already within Braze using the alias, {% raw %}`shopify_customer_id_{{storefront_domain}}` `shopify_email`, or `shopify_phone`, {% endraw %} then we’ll update the existing user profile. If those aliases do not exist within Braze, we will create a new user profile. Note that it is possible for a user’s data (e.g., city) to differ across multiple Shopify stores for the same user; in such cases, Braze will always update the user profile from the store with the most recent activity.
```
Comment (lydia-xie) [inline _docs/_hidden/private_betas/multiple_stores.md:60]:
```suggestion
If the user’s Shopify customer ID, email address, or phone number exists already within Braze using the alias, {% raw %}`shopify_customer_id_{{storefront_domain}}`, `shopify_email`, or `shopify_phone`, {% endraw %} then we’ll update the existing user profile. If those aliases do not exist within Braze, we will create a new user profile. Note that it is possible for a user’s data (e.g., city) to differ across multiple Shopify stores for the same user. In such cases, Braze will always update the user profile from the store with the most recent activity. 
```
Adding a comma here 🤔

**Review by lydia-xie — APPROVED** [review]

**Pull request merged**